### DISCUSSION TITLE
blog/highlighting-adventures

### DISCUSSION THREAD

**Discussion by giscus** [CodeMyst/myst.rs#24] [Announcements]
# blog/highlighting-adventures

CodeMyst's personal portfolio and blog.

https://myst.rs/blog/highlighting-adventures

**Comment by haaarshsingh** [comment]
Interesting. For React we have options such as [React Textarea Code Editor](https://uiwjs.github.io/react-textarea-code-editor/) which make it insanely easy to incorporate something similar in React websites. Also somewhat customisable :)

**Comment by CodeMyst** [comment]
@harshhhdev yeah, but what does it use for actual highlighting behind the scenes? the point of this article was to focus on **good** syntax highlighting using advanced methods, and not something lighter like highlight.js for example.
